os2forms_nemlogin_openid_connect.module

```diff
@@ -62,8 +63,30 @@ function os2forms_nemlogin_openid_connect_theme(array $existing, string $type, s
     'os2forms_nemlogin_openid_connect_local_test_users' => [
       'variables' => [
         'users' => NULL,
+        'plugin' => NULL,
         'query' => NULL,
       ],
     ],
   ];
 }
+
+/**
+ * Implements hook_os2web_nemlogin_auth_provider_info_alter().
+ *
+ * @phpstan-param array<string, mixed> $definitions
+ */
```

Comment:
A comment stating what this function actually does might come in handy. A note on where the definition “OpenIDConnect” comes from will also be useful.

Reply:
Added comment.